app/src/main/java/com/example/moviedb/ui/adapter/MovieAdapter.kt

```diff
@@ -12,7 +12,7 @@ import com.example.moviedb.databinding.ItemMovieBinding
 import com.example.moviedb.ui.utils.ImageUtils
 
 class MovieAdapter(
-    private val movies: List<Movie>
+    private val movies: MutableList<Movie>
```

Comment:
có thể cài đặt giá trị default cho thuộc tính 
```suggestion
    private val movies: MutableList<Movie>= mutableListOf()
```

Reply:
như thế có thể khởi tạo từ lúc không chưa có movies 
``` Kotlin
val adapter = MovieAdapter() 
```
Với những trường hợp mà khi khởi tạo Adapter, ta luôn luôn KHÔNG truyền movies tại thời điểm ban đầu mà truyền qua hàm như addMovies 
thì không nên để thuộc tính tại hàm khởi tạo 
``` Kotlin
class Adapter {
    private val movies = mutableListOf()
}
```